From 6866f29b4b62ca172838aa8f5aa59d409ae0d76c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:11:03 -0400
Subject: [kernel] tty: Add C_CMSPAR(tty)

Message-id: <1410887463-20674-592-git-send-email-dzickus@redhat.com>
Patchwork-id: 95406
O-Subject: [RHEL7 PATCH 591/591] tty: Add C_CMSPAR(tty)
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit e58b57a354cade526266f60d3d936eed7c421c90
Author: Andrew Lunn <andrew@lunn.ch>
Date:   Sun Dec 29 19:23:16 2013 +0100

    tty: Add C_CMSPAR(tty)

    Add the missing C_CMSPAR(tty) macro.

    Signed-off-by: Andrew Lunn <andrew@lunn.ch>
    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/tty.h b/include/linux/tty.h
index 8780bd2..4e59c42 100644
--- a/include/linux/tty.h
+++ b/include/linux/tty.h
@@ -134,6 +134,7 @@ struct tty_bufhead {
 #define C_CLOCAL(tty) _C_FLAG((tty), CLOCAL)
 #define C_CIBAUD(tty) _C_FLAG((tty), CIBAUD)
 #define C_CRTSCTS(tty) _C_FLAG((tty), CRTSCTS)
+#define C_CMSPAR(tty) _C_FLAG((tty), CMSPAR)
 
 #define L_ISIG(tty) _L_FLAG((tty), ISIG)
 #define L_ICANON(tty) _L_FLAG((tty), ICANON)
-- 
1.7.1